pom.xml

```diff
@@ -56,6 +56,21 @@
                     <validateOnly>true</validateOnly>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-source-plugin</artifactId>
+                <version>2.3</version>
+                <executions>
+                    <execution>
+                        <id>attach-sources</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>jar-no-fork</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
```

Comment:
This should actually go in the gwtbootstrap3-parent project pom.xml.  Please revert this change and I will refactor the pom's.  Thanks!